packages/apptive_grid_form/lib/src/apptive_grid_form.dart

```diff
@@ -795,3 +590,389 @@ class ApptiveGridFormDataState extends State<ApptiveGridFormData> {
+                  padding: padding,
+                  child: Builder(
+                    builder: (context) {
+                      final customBuilder =
+                          componentBuilder?.call(context, component);
+                      if (customBuilder != null) {
+                        return customBuilder;
+                      } else {
+                        return componentWidget;
+                      }
+                    },
+                  ),
+                ),
+              );
+            }
+          } else {
+            return Padding(
+              padding: padding,
+              child: Align(
+                alignment: buttonAlignment,
+                child: Builder(
+                  builder: (_) {
+                    if (isSubmitting) {
+                      return Column(
+                        mainAxisSize: MainAxisSize.min,
+                        crossAxisAlignment: CrossAxisAlignment.stretch,
+                        children: [
+                          const TextButton(
+                            onPressed: null,
+                            child: Center(
+                              child: CircularProgressIndicator.adaptive(),
+                            ),
+                          ),
+                          if (progress != null)
+                            Padding(
+                              padding: padding,
+                              child: SubmitProgressWidget(progress: progress!),
+                            ),
+                        ],
+                      );
+                    } else if (!hideButton) {
+                      return ElevatedButton(
+                        onPressed: submitForm,
+                        child: Text(
+                          buttonLabel ??
+                              data.properties?.buttonTitle ??
+                              localization.actionSend,
+                        ),
+                      );
+                    } else {
+                      return const SizedBox();
+                    }
+                  },
+                ),
+              ),
+            );
+          }
+        },
+      ),
+    );
+  }
+}
+
+/// Displays an error the occured while submitting
+class FormErrorWidget extends StatelessWidget {
+  /// Creates a new error widget
+  const FormErrorWidget({
+    super.key,
+    required this.error,
+    required this.padding,
+    required this.didTapBackButton,
+    this.scrollController,
+    this.formData,
+  });
+
+  /// The error being displayed
+  final dynamic error;
+
+  /// Padding of the Items in the Form. If no Padding is provided a EdgeInsets.all(8.0) will be used.
+  final EdgeInsetsGeometry padding;
+
+  /// Triggers when the user taps the back button
+  final Function() didTapBackButton;
+
+  /// Optional ScrollController for the Form
+  final ScrollController? scrollController;
+
+  /// The current form data
+  final FormData? formData;
```

Comment:
I don't see where this is used. I guess we should therefor remove it

Reply:
done